client/js/common.ts

```diff
@@ -7,7 +7,7 @@ async function checkStatus(response: Response) {
 		return response;
 	}
 	else {
-		return new Error((await response.json()).error);
+		throw new Error((await response.json()).message);
```

Comment:
Sorry, just realized this: the error message needs to remain the `error` property of the JSON object to remain compatible with the previous code that uses the convention `{"success": true}` for successful actions and `{"error": "An error message"}` for errors.